release-notes/4.0.0.md

```diff
@@ -0,0 +1,54 @@
+# Cucumber-ruby 4.0.0 release notes
+
+This is the first stable Cucumber releasse of the 4.x serie. It's the first stable release since 2 years and there's pretty big changes since 3.1.2, let's have a look at the biggest changes introduced by this release.
```

Comment:
```suggestion
This is the first stable Cucumber release of the 4.x series. It's the first stable release since 2 years and there's pretty big changes since 3.1.2, let's have a look at the biggest changes introduced by this release.
```

Reply:
thanks for the typo fix :)